iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractIdentifiable.java

```diff
@@ -80,20 +81,34 @@ public String getNameOrId() {
 
     @Override
     public void addAlias(String alias) {
-        addAlias(alias, null);
+        addAlias(alias, false);
```

Comment:
This could be considered as a breaking change because the default behaviour is not the same anymore

Reply:
I don't think it changes the default behaviour, before my change, this method behaved as if unicity was not ensured: an exception was thrown when an already existing alias was passed